sdks/python/test-suites/tox/py36/build.gradle

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Unit tests for Python 3.6
+ */
+
+plugins { id 'org.apache.beam.module' }
+applyPythonNature()
+
+toxTask "testPython36", "py36"
```

Comment:
How come there's no py36-gcp?

Reply:
py36 cython and gcp suites are being worked on and will be added.

Reply:
Specifically, in https://github.com/apache/beam/pull/8036

Reply:
Note, this PR is target on reorganizing tasks to make parallel work, not enable new test suite, which requires more manual effort to investigate failure and do the fix. @RobbeSneyders has more context on new suite work.